requirements/app.txt

```diff
@@ -57,6 +57,23 @@ entrypoints==0.3
     # via altair
 filelock==3.0.12
     # via tldextract
+flask==2.0.1
```

Comment:
Something seems to have changed in pip-tools, so that pip-compile lists the main package first, then the sub-packages (e.g. flask before flask-classful).

Crucial: Are you using a later version of pip-tools or me? (probably you as I recall your system is newly-installed):

This is mine:
```
± pip list | grep pip 
pip                    21.1.3
pip-tools              6.0.1
```

Reply:
Indeed I am using a later version of pip-tools:
```
pip                           21.1.3
pip-tools                     6.2.0
```
[pip-tools 6.2](https://github.com/jazzband/pip-tools/blob/master/CHANGELOG.md) indeed changed the ordering. It also logs which Python version was used during compile (but I did not commit that line change). What is the best way forward here: shouldn't our makefile pin versions, actually?

Reply:
Yes. Thanks for making that happen.